crates/core/src/surfnet/svm.rs

```diff
@@ -110,6 +111,7 @@ impl SurfnetSvm {
                 transactions_queued_for_finalization: VecDeque::new(),
                 signature_subscriptions: HashMap::new(),
                 tagged_profiling_results: HashMap::new(),
+                updated_at: Utc::now().timestamp_millis() as u64,
```

Comment:
Currently, we're updating this value when we set an account or send a transaction. So technically, not all RPC requests will reset this value.

If I do a bunch of `getBalance` RPC requests to my network, my network will still shut down, even though I'm using it.

However, we can only set this value on functions that are `&mut self`, and we don't want all of these functions to have to write to the `SurfnetSvm`.

For now, I'm okay with this, with one exception. Can you set the `updated_at` on every fn in this file that is `&mut self`?


In the future, I think we can add some middleware to the RPC which will reset this value at every request. And in the future I think this `updated_at` field won't live on the `SurfnetSvm`